readme.md

```diff
@@ -214,6 +214,28 @@ ow(5, ow.number.is(x => greaterThan(10, x)));
 //=> ArgumentError: Expected `5` to be greater than `10`
 ```
 
+#### validate(fn)
+
+Use a custom validation object. The difference with `is` is that the function should return a validation object which allows more flexibility.
+
+```ts
+ow(1, ow.number.validate(x => ({
+	validator: x > 10,
+	message: `Expected value to be greater than 10, got ${x}`
+})));
+//=> ArgumentError: (string) Expected value to be greater than 10, got 1
+```
+
+You can also pass in a function as `message` value which accepts the label as argument.
+
+```ts
+ow(1, 'input' ow.number.validate(x => ({
+	validator: x > 10,
+	message: label => `Expected ${label} to be greater than 10, got ${x}`
+})));
+//=> ArgumentError: Expected number `input` to be greater than 10, got 1
+```
+
```

Comment:
Maybe also mention that this can be useful for creating your own reusable validations that could potentially be in a separate npm package. And example of this would be useful.

Reply:
Yes, I'm working on a `ow-date` package which uses `date-fns` under the hood which can do much more advanced date checking then the built-in ones.

I was wondering if we should release this under `@ow/date` or something? Or do you rather prefer it to just be `ow-date`?

Reply:
Actually when looking at it again, the `ow-date` package is not really relevant in this section as it's really a separate predicate, not just a validation function.

Reply:
> I was wondering if we should release this under @ow/date or something? Or do you rather prefer it to just be ow-date?

I think `ow-date` is fine. I doubt we're gonna have conflicts with `ow-*`.